content/en/continuous_integration/setup_pipelines/github.md

```diff
@@ -43,6 +43,16 @@ The [Pipelines][4] and [Pipeline Executions][5] pages populate with data after t
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+## Disabling GitHub Actions tracing
+
+To disable the CI Visibility GitHub Actions integration make sure the GitHub app is no longer subscribed to the
+workflow job and workflow run events. To remove the events:
+
+1. Go to the [GitHub Apps][6] page.
+2. Click **Edit > Permission & events** on the relevant Datadog GitHub App (if you have multiple apps, you will have to repeat the process for each).
+3. Scroll down to the **Subscribe Events** and make sure that **Workflow job** and **Workflow run** are not selected.
```

Comment:
I'm trying to follow along with a quick tester github app. Is this what you're referring to?
<img width="781" alt="Screen Shot 2022-03-29 at 10 11 39 AM" src="https://user-images.githubusercontent.com/605271/160667874-b9c5d67a-9db0-4078-a369-5cc1db99aa92.png">

if so, we should match their UI text exactly to be this:
```suggestion
3. Scroll to the **Subscribe to events** section, and make sure that **Workflow job** and **Workflow run** are not selected.
```